class-zoninator-rest-api-controller.php

```diff
@@ -0,0 +1,565 @@
+<?php
+
+
+class Zoninator_Rest_Api_Controller {
```

Comment:
There's some inconsistency in the method names in the controller. We use the generic `items` in some cases and also more explicit `zone` + `posts` in others. More explicit/descriptive names would be better.